v3/src/components/mosaic-tile-row.tsx

```diff
@@ -13,13 +14,14 @@ import "./mosaic-tile-row.scss"
  */
 interface IMosaicTileRowProps {
   row: IMosaicTileRow
+  content?: IDocumentContentModel
   getTile: (tileId: string) => ITileModel | undefined
 }
-export const MosaicTileRowComponent = ({ row, getTile }: IMosaicTileRowProps) => {
+export const MosaicTileRowComponent = ({ content, row, getTile }: IMosaicTileRowProps) => {
```

Comment:
This should probably be an `observer()` as well, so that it re-renders if `row.root` changes, for instance.